.gitignore

```diff
@@ -1,2 +1 @@
 /node_modules/
```

Comment:
Man, you still haven't configure your code editor to keep or add the last line empty on files…

Reply:
not my laptop ^^

Reply:
Your suggestion isn't very inclusive ;)
Another solution: https://editorconfig.org/

Reply:
`.editorconfig` init